docs/sdk/0-core-feature/1_web3names.md

```diff
@@ -0,0 +1,54 @@
+---
+id: web3names
+title: Web3 names
+---
+import CodeBlock from '@theme/CodeBlock';
+import Example1 from '!!raw-loader!../../../code-examples/core_features/web3names/1_web3name.ts';
+import Example2 from '!!raw-loader!../../../code-examples/core_features/web3names/2_web3name.ts';
+import Example3 from '!!raw-loader!../../../code-examples/core_features/web3names/3_web3name.ts';
+
+Web3 names are user-friendly aliases for a KILT DID.
+They serve the same purpose that domain names do for IP addresses: who knows what are all the IP addresses under the `google.com` domain name? 🤷🏽‍♀️
+
+Each Web3 name is globally unique, and is composed of a sequence of a minimum of 3 to a maximum of 32 characters taken from a specific character set to enhance human redability and reduce the changes of "two Web3 names looking the same, despite being different".
```

Comment:
This Web3 name is unique for the KILT Blockchain? Someone can make a new chain and create a new Web3 name.

Reply:
https://github.com/KILTprotocol/docs/pull/62/commits/0cc2f751f9bc26c540550abf32c687eef81da558